zinc/src/test/scala/sbt/inc/AuxiliaryClassFilesSpec.scala

```diff
@@ -0,0 +1,29 @@
+package sbt.inc
+
+import sbt.io.IO
+import xsbti.compile.AuxiliaryClassFiles
+
+import java.nio.file.Path
+
+class AuxiliaryClassFilesSpec extends BaseCompilerSpec {
+  it should "allow client to add define their own auxiliary class files" in {
```

Comment:
```suggestion
  it should "allow client to define their own auxiliary class files" in {
```

Reply:
this test does not exist anymore because it was useless